sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -52,10 +52,6 @@ public class StorageBlobClient: PipelineClient, PagedCollectionDelegate {
         }
     }
 
-    private lazy var manager: URLSessionTransferManager = {
```

Comment:
Ultimately I want to revisit this, from a user's perspective I'm working to avoid exposing the transfer manager at all, and just run those operations through the public API surface of StorageBlobClient. I went the other way in my branch, promoting the TransferDelegate to a (public) property of the Client and hiding the manager as private. I understand why this was moved to the Options, but ultimately I think what we are going to have to do is:

1. Serialize the transfer-specific Options (e.g. DownloadBlobOptions) associated with a transfer to CoreData so that we can access them if the app is restarted, and
2. Also store the subset of StorageBlobClient properties / Options needed to determine whether any given Client is "suitable" to be used for a transfer. That might allow us to get rid of the "downloader/uploader" property on the Transfer, and instead just keep a record of all Clients created, and if that list changes, determine if there is a "suitable" client for any transfers without an owning client, and assign & resume them.

There might be other approaches that would work better, but for now this is the way I'm thinking about avoiding the need for us to try to re-create a Client for a transfer that was not completed.

Reply:
Now that we have some implementation, we should use our time Wednesday to revisit this problem with the Android team as well. When we discussed it before it was all hypothetical.